### PR TITLE
enhancement: embedify all of the meta commands

### PR DESCRIPTION
**Pull request by onerandomusername**
Old responses:
image

New responses:
image

### CODE REVIEW

**Review by Shivansh-007 — APPROVED** [review]
Yes

**Comment by onerandomusername** [comment]
This pr should wait until GH-75 is merged as it will cause merge conflicts, and I would rather resolve them here than on that pr.

**Comment by onerandomusername** [comment]
Actually, the conflicts are easy enough to fix there, so I'll merge this now.

**Pull request merged**
